### ISSUE TITLE
Your location is not supported by google-generativeai at the moment.

### ISSUE THREAD

**Issue by aaaa-zhen** [svpino/alloy-voice-assistant#8]
_ValueError: Your location is not supported by google-generativeai at the moment. Try to use ChatVertexAI LLM from langchain_google_vertexai._

it's there anyway to use another API ?

**Comment by svpino** [comment]
Yes, you can use GPT-4o. That code is commented out. Uncomment it and you are good to go.
